### DISCUSSION TITLE
[Feature] face detection, change feature photo => list of possible thumbs, ordered by a "quality criteria"

### DISCUSSION THREAD

**Discussion by aisbergde** [immich-app/immich#6348] [Feature Request]
### The feature

I have seen in feature descriptions, that in the future selecting the best match for a person will be improved.
Would it be possible, to add some improvements also to already detected faces?
I could imagine the following:
When I click on "change feature photo", then I get a list of sorted feature photos, sorted by using some "quality criteria":
- distance to the average face vector
- thumb quality (size, sharpness, ...).
 
If the "select thumb algorithm" will be improved, the same code could be reused to order the possible thumbs.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Closing in favour of #10051.
